app/components/slack-window.js

```diff
@@ -44,7 +44,7 @@
 			var that = this;
 
 			win.addEventListener('contextmenu', function handleContextMenu (evt) {
-				SlackContextMenu.handleRightClick(that.getWindow(), evt.target, evt.x, evt.y);
+				SlackContextMenu.handleRightClick({window: that.getWindow(), target: evt.target, x: evt.x, y: evt.y, teamUrl: that.props.team.team_url});
```

Comment:
This line is a bit lengthy. Maybe we can create a `var params` with a couple of line feeds on the previous line to shorten it?

```js
var params = {
    target: evt.target, x: evt.x, y: evt.y,
    window: that.getWindow(), teamUrl: that.props.team.team_url
};
SlackContextMenu.handleRightClick(params);
```